Source/Android/app/src/main/res/values/strings.xml

```diff
@@ -858,4 +859,17 @@ It can efficiently compress both junk data and encrypted Wii data.
     <string name="compression_lzma2">LZMA2 (slow)</string>
     <string name="compression_zstandard">Zstandard (recommended)</string>
 
+    <!-- About dialog -->
+    <string name="about_dolphin">Dolphin</string>
+    <string name="wark" translatable="false">WARK WARK WARK</string>
```

Comment:
I think it makes sense to localize onomatopoeia.

Reply:
Huh, forgot that's a thing